### ISSUE TITLE
Please add TPU kernels for tf.linalg.[[s]log]det

### ISSUE THREAD

**Issue by sharvil** [tensorflow/tensorflow#33402]
<em>Please make sure that this is a feature request. As per our [GitHub Policy](https://github.com/tensorflow/tensorflow/blob/master/ISSUES.md), we only address code/doc bugs, performance issues, feature requests and build/installation issues on GitHub. tag:feature_template</em>

**System information**
- TensorFlow version (you are using): 1.15
- Are you willing to contribute it (Yes/No): No

**Describe the feature and the current behavior/state.**
Adding determinant kernels (along with gradients) will allow us to train Normalizing Flow models on TPUs.

**Will this change the current api? How?**

**Who will benefit with this feature?**

**Any Other info.**

**Comment by MokkeMeguru** [comment]
I want to support for Tensorflow == 2.2.0, too.

**Comment by tensorflowbutler** [comment]
Hi There,

 We are checking to see if you still need help on this, as you are using an older version of tensorflow which is officially considered end of life . We recommend that you upgrade to the latest 2.x version and let us know if the issue still persists in newer versions. Please open a new issue for any help you need against 2.x, and we will get you the right help. 

 This issue will be closed automatically 7 days from now. If you still need help with this issue, please provide us with more information.

**Comment by sharvil** [comment]
AFAIK, this feature request has not been implemented.

**Comment by sharvil** [comment]
Nice.
